src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -74,7 +74,18 @@ public void process(Objection objection, String httpRequestId)
 
         // TODO update status to CHIPS_SENT
 
-        // TODO OBJ-172 send internal email
+        // send internal email
+        try {
+            CompanyProfileApi companyProfile = this.companyProfileService.getCompanyProfile(objection.getCompanyNumber(), httpRequestId);
+            emailService.sendObjectionSubmittedDissolutionTeamEmail(companyProfile.getCompanyName(), companyProfile.getJurisdiction(), objection, httpRequestId);
+        } catch (Exception e) {
+            logMap = new HashMap<>();
```

Comment:
Then both email calls could go in the same try.. catch block then add log info the email being sent - customer /dissolution and then have a generic exception message. Alternatively set a string varaible for the email being sent and make that aprt of the exception message so that it is still tailored.

Reply:
Yep, agree with @mattch1 in that we could have one try/catch block. As long as the exception stack trace is present in the logged error message (it is/should be) then we have the line number and we'll know what e-mail failed. This is kind of the intention of Java try/catch blocks - code for the sunny days scenario and deal with all the error cases below.

Reply:
I'm not sure about why you would cause a need for someone to look at the code by having generic logs when you could just be specific, as these logs will be for the support staff, not us. It was Mark who originally suggested keeping these try/catch blocks separate to make this easier. If you guys are sure though I don't mind changing it!

Reply:
ok i'm happy to go with a generic catch fr both calls, but keep the TODO INTERNAL_EMAIL_SENT status change in between them, we'll need to do that in future

Reply:
Thinking about this some more, I'd like to keep them separate as this will give us the ability to set separate error status' indicating what type of email failed and will also allow support team to re-run them independently in future